### DISCUSSION TITLE
Which version is LTS?

### DISCUSSION THREAD

**Discussion by huang-x-h** [apache/apisix#6851] [Q&A] [answered]
At present, the LTS version has 2.10.* and 2.12.*, does 2.13.0 is LTS?

And how long the tow or three LTS version to matain?  Which version is recommended?

**Comment by leslie-tsang** [comment, marked as answer]
Hello, there are **<ins>two</ins>** LTS versions `2.10` and `2.13`.

> Which version is recommended?

`2.13` is recommended

**Reply by huang-x-h** [reply]
ok, i get it

**Reply by tokers** [reply]
@leslie-tsang Where can users find this information?

**Reply by juzhiyuan** [reply]
https://apisix.apache.org/downloads

Our Download page only shows the latest LTS version. Another channel is CHANGELOG, and it will show `This is an LTS version,` but I noticed since V2.12.1, there has been no tip for this. see https://github.com/apache/apisix/blob/release/2.13/CHANGELOG.md#2121

cc @leslie-tsang @navendu-pottekkat @SkyeYoung Any ideas on this? (How to guide users quickly find LTS version)

**Reply by SkyeYoung** [reply]
I created a [repository ](https://github.com/SkyeYoung/apisix-versions) for parsing CHANGELOG and have a website available. It is very easy to find the LTS version inside. 😂

**Reply by juzhiyuan** [reply]
![image](https://user-images.githubusercontent.com/2106987/164144754-7a013ce8-6def-4221-aa65-509d71f8e059.png)

**Reply by pottekkat** [reply]
Showing the latest LTS version is the ideal scenario right? We would always want the users using the latest version of APISIX.

**Reply by tokers** [reply]
Right, I think the latest LTS version should be shown in an easy, obvious way, but we still should have a place to see all LTS versions.

**Reply by tzssangglass** [reply]
> Right, I think the latest LTS version should be shown in an easy, obvious way, but we still should have a place to see all LTS versions.

+1

**Reply by pottekkat** [reply]
Yes, makes sense. +1

**Reply by huang-x-h** [reply]
LTS versions have maintenance time, end of life?

**Reply by tzssangglass** [reply]
Basically, it is half a year
